docs/model/simulation_outputs.rst

```diff
@@ -318,7 +318,7 @@ A commit hash number is provided at the end of the report in order to prevent th
 It includes several tables with project data, simulation settings, the various demands supplied by the user, the various components of the system and the optimization results such as the energy flows and the costs.
 The report also provides several plots which help to visualize the flows and costs.
 
-Please, refer to the `report section <https://mvs-eland.readthedocs.io/en/latest/Installation.html#generate-pdf-report-or-an-app-in-your-browser-to-visualise-the-results-of-the-simulation>`__ for more information on how to setup and use this feature, or type
+Please, refer to the :ref:`report section <pdf-report-commands>` for more information on how to setup and use this feature, or type
```

Comment:
or this one

Reply:
These are the same links, and it is probably related to your "Installation section which is empty locally. Did you try to refresh your browser?